### PR TITLE
Stale Benefit Intake Submission Logging

### PR DESCRIPTION
**Pull request by wayne-weibel**
## Summary

refactor benefits_intake_status_job to report on stale submissions

## Related issue(s)

https://github.com/department-of-veterans-affairs/va.gov-team/issues/85842

## Testing done

- [x] *New code is covered by unit tests*
- create a form_submission and related attempt
- set created_at time of attempt to a day in distant past
- run the benefits_intake_status_job
- there should be a statsd increment for the form type with 'stale' and Rails log

## What areas of the site does it impact?

any app using LH BenefitsIntake API and the FormSubmission, FormSubmissionAttempt tracking

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution

### CODE REVIEW

File: app/sidekiq/benefits_intake_status_job.rb
Comment (mjknight50) [inline app/sidekiq/benefits_intake_status_job.rb:1]: Can you please link somewhere in this file to the corresponding datadog dashboard?

File: app/sidekiq/benefits_intake_status_job.rb [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
+require 'lighthouse/benefits_intake/service'
 
 class BenefitsIntakeStatusJob
   include Sidekiq::Job
 
   sidekiq_options retry: false
 
   STATS_KEY = 'api.benefits_intake.submission_status'
-  MAX_BATCH_SIZE = 1000
+  STALE_SLA = Settings.lighthouse.benefits_intake.report.stale_sla || 10
```
[the remark this answers is not in the document]
Comment (wayne-weibel) [reply app/sidekiq/benefits_intake_status_job.rb:11]: yes

File: spec/sidekiq/benefits_intake_status_job_spec.rb [hunk 2]
```diff
@@ -65,28 +65,122 @@
...
+          .with(uuids: batch_uuids).and_return(response)
+
+        status_job.perform
+
+        pending_form_submissions.each do |form_submission|
+          expect(form_submission.form_submission_attempts.first.reload.aasm_state).to eq 'pending'
+        end
+      end
+
+      # 'updating the form submission status'
```
[the remark this answers is not in the document]
Comment (wayne-weibel) [reply spec/sidekiq/benefits_intake_status_job_spec.rb:167]: yes, this makes it easier to find/see where the blocks end

**Review by mjknight50 — APPROVED** [review]
GTG

**Review by ryan-mcneil — APPROVED** [review]
🚢

**Pull request merged**